### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by MehdizadeMilad**

### CODE REVIEW

**Comment by eggplantzzz** [comment]
Thanks for suggesting this change to the docs! I changed the PR to be against the `develop` branch as that is the branch we use to merge new work.

File: packages/contract/README.md [hunk 1]
```diff
@@ -308,7 +308,7 @@ instance.getValue.call().then(function(val) {
 });
 ```
 
-Even more helpful, however is we *don't even need* to use `.call` when a function is marked as `view` or `pure`, because `@truffle/contract` will automatically know that that function can only be interacted with via a call:
+Even more helpful, however we *don't even need* to use `.call` when a function is marked as `view` or `pure`, because `@truffle/contract` will automatically know that that function can only be interacted with via a call:
```
Comment (eggplantzzz) [inline packages/contract/README.md:311]: I think we want to leave in the "is" in this case as this part of the sentence requires a verb to be grammatically correct and to say exactly *what* is more helpful.

**Review by cds-amal — COMMENTED** [review]
I agree with @eggplantzzz though a comma after "however" may be helpful.

File: packages/contract/README.md (hunk 1 above)
Comment (haltman-at) [reply packages/contract/README.md:311]: The `is` isn't necessarily here; the verb phrase is that we don't even need etc.  But there should be a comma after the however.

Comment (eggplantzzz) [reply packages/contract/README.md:311]: If you write "Even more helpful, ..." it seems that you are referencing something that was just mentioned. That is not the case here. This text means to say, "hey, here is something else that is helpful".

Comment (haltman-at) [reply packages/contract/README.md:311]: ? This is referencing the `.call` functionality that was described immediately preceding.

Comment (eggplantzzz) [reply packages/contract/README.md:311]: Yeah, it is referencing `.call` which is goes on to describe afterwards.

**Comment by eggplantzzz** [comment]
So I think we should close this, any opposed?

**Comment by cds-amal** [comment]
no opposition 👍

**Pull request closed without merging**